streams/src/test/java/org/apache/kafka/streams/integration/EosIntegrationTest.java

```diff
@@ -93,7 +98,7 @@
         Utils.mkProperties(Collections.singletonMap("auto.create.topics.enable", "false"))
     );
 
-    private static String applicationId;
+    private String applicationId;
```

Comment:
I realized that the test setup was actually incorrect. We try to use a unique `application.id` per test method and with parallel test execution the variable cannot be `static`